.github/workflows/test.yaml

```diff
@@ -54,14 +49,46 @@ jobs:
           echo "Failed to resolve after 15 seconds"
           exit 1
 
+      - name: Enable KVM
```

Comment:
Enable hardware acceleration so our emulators are faster 🚗

Reply:
Wow we can just do this? Amazing

Reply:
Sick!